osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -141,11 +141,10 @@ private void updateRate()
         {
             if (sourceClock == null) return;
 
-            sourceClock.Rate = 1;
+            sourceClock.Rate = UserPlaybackRate.Value;
```

Comment:
Use `sourceClock.ResetSpeedAdjustments()`.

Reply:
even though it's effectively going to be overridden by the `UserPlaybackRate`?

Reply:
Well it's not, only the tempo will, but the frequency won't be reset.